ui/src/components/Homepage/index.jsx

```diff
@@ -65,9 +65,8 @@ export default class HomePage extends Component {
           </Typography>
         </Hidden>
         <Typography variant="h5">
-          {
-            "A task execution framework that supports Mozilla's continuous integration and release processes."
-          }
+          &quot; A task execution framework that supports Mozilla&apos;s
```

Comment:
```suggestion
          &quot;A task execution framework that supports Mozilla&apos;s
```
and is the `&apos;` really necessary??

Reply:
Its to differentiate between `Mozillas` and `Mozilla's`

Reply:
I mean instead of just using the `'` character.

Reply:
It threw a error on console .